unit.js

```diff
@@ -0,0 +1,66 @@
+function Unit(name){
+  this.MAXHEALTH = 100;
+  this._name = name;
+  this._health = this.MAXHEALTH;
+}
+
+Unit.prototype._level = 1;
+Unit.prototype._xp = 0;
+
+Unit.prototype.getName = function() {
+  return this._name;
+}
+
+Unit.prototype.getLevel = function() {
+  return this._level;
+}
+
+Unit.prototype.getHealth = function() {
```

Comment:
No need for public getters for health and xp, you should be using them only within Unit classes and its ancestors